pub/README.md

```diff
@@ -4,14 +4,12 @@ Dart (pub) support for [`dependabot-core`][core-repo].
 
 ### Limitations
 
- - No support for updating git-dependencies
-   * `dart pub` in general doesn't read versions from git, so upgrade logic is limited.
-   * Some variant of support for updating git-dependencies could be added in the future.
+ - Limited updating of git-dependencies
+   * `dart pub` in general doesn't read versions from git, so upgrade logic is limited to upgrading to what the 'ref' is pointing to.
```

Comment:
Maybe state that it won't upgrade to a new branch, or a new tag, or a new revision if that is specified in `pubspec.yaml`.
But will upgrade resolved-ref in `pubspec.lock` to new revision from specified branch or HEAD (if no branch is specified).